runLocally.sh

```diff
@@ -21,7 +21,7 @@ set -e
 shopt -s nullglob
 
 function confirm() {
-  declare -l answer # -l converts the value to lower-case before it's assigned
+  declare answer
   read -r -p "$1 [y/N]: " answer
```

Comment:
The capital "N" has nothing to do with the question whether the accepted answer is case sensitive or not. The capitalized letter signifies the default answer.

In other words, when you see `[y/N]` it should be clear what your options are and what the default/safe/recommended option is. Knowing that allows you to take a shortcut and just hit Enter without having to type the default answer.

I copied this behavior from the `dnf` package manager because I find it useful and I thought it's a convention for linux CLIs. But I might be wrong. The only source supporting this I could find retrospectively is this: https://ux.stackexchange.com/a/40445/43532.

Reply:
Sorry for that. The idea is pretty cool. Thanks for the explanation